### PR TITLE
Extend user documentation with a warning about tools being Linux-only

### PR DESCRIPTION
**Pull request by sbastrakov**

### CODE REVIEW

**Review by PrometheusPi — DISMISSED** [review]
great addition to the documentation :+1:

File: USAGE.rst [hunk 1]
```diff
@@ -4,6 +4,13 @@
 
    You need to have an :ref:`environment loaded <install-profile>` (``source $HOME/picongpu.profile``) that provides all :ref:`PIConGPU dependencies <install-dependencies>` to complete this chapter.
 
+.. warning::
+
+   PIConGPU source code is portable and can be compiled on all major operating systems.
+   However, helper tools like ``pic-create`` and ``pic-build`` described in this section rely on Linux utilities and thus are not expected to work on other platforms out-of-the-box.
```
Comment (psychocoderHPC) [inline USAGE.rst:10]: `Linux.utilities` should be `Linux utilities`

**Review by psychocoderHPC — CHANGES_REQUESTED** [review]
small typo

**Comment by psychocoderHPC** [comment]
I will stop the CI by hand that you can fix the typo

**Comment by psychocoderHPC** [comment]
If we merge first #3452 and you rebase after the merge, you can use the power of the CI action `CI:no-compile`

File: USAGE.rst (hunk 1 above)
Comment (sbastrakov) [reply USAGE.rst:10]: I am not sure what do you mean, to me it seems to already be "Linux utilities". Could you elaborate?

**Comment by sbastrakov** [comment]
@psychocoderHPC I am not sure what do  you mean in the change request. I rebased on the current dev without changing anything to test skipping CI

File: USAGE.rst (hunk 1 above)
Comment (psychocoderHPC) [reply USAGE.rst:10]: There was a dot in between the words. But looks like it is now fixed

**Review by psychocoderHPC — APPROVED** [review]

**Pull request merged**